opencti-platform/opencti-front/lang/front/en.json

```
@@ -1546,12 +1550,14 @@
   "if needed": "if needed",
   "If needed, you can set a": "If needed, you can set a",
   "If the CSV mapper is configured with \"Use default markings definitions of the user\", the default markings of the user responsible for data creation are applied to the ingested entities. Otherwise, you can choose markings to apply.": "If the CSV mapper is configured with \"Use default markings definitions of the user\", the default markings of the user responsible for data creation are applied to the ingested entities. Otherwise, you can choose markings to apply.",
+  "If the email address you entered is associated with an account, you’ll receive a confirmation email with a reset code shortly.": "If the email address you entered is associated with an account, you’ll receive a confirmation email with a reset code shortly.",
```

```suggestion
  "If the email address you entered is associated with an account, you’ll receive a confirmation email with a reset code shortly.": "If the email address you entered is associated with an account, you will receive a confirmation email with a reset code shortly.",
```

No word abbreviation on end users texts please.